### PR TITLE
[FEAT] Add support for codium

### PR DESCRIPTION
**Pull request by subins2000**
If there's no vscode, then use `codium`

### CODE REVIEW

File: packages/connector/src/actions.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ function cloneAndOpenRepo(cloneURL, tab_id) {
         });
       });
   } else {
-    const command = `git clone ${cloneURL} ${localPath} && code ${localPath}`;
+    const command = `git clone ${cloneURL} ${localPath} && $(which code || which codium) ${localPath}`;
```
Comment (haxzie) [inline packages/connector/src/actions.js:98]: If the initial program doesn't exist, the command fails. Can you check this once if it is working? Tried it in my PC doesn't seem to work. Instead this should work.
```suggestion
    const command = `git clone ${cloneURL} ${localPath} && code ${localPath} || codium ${localPath}`;
```

What do you think?

**Review by haxzie — CHANGES_REQUESTED** [review]
Command fails due to `$` please check if the `$` is required. Thank you :)

**Comment by haxzie** [comment]
Ah, sorry my bad. This does work. Thank you :)

**Review by haxzie — APPROVED** [review]

**Comment by haxzie** [comment]
What do you think about adding a configuration command along with the connecter where users can simply provide an editor command? something like..
```bash
open-with-code --editor code
# or
open-with-code --editor codium/sublime/etc
```

any thoughts?

**Comment by haxzie** [comment]
#2  Added support for codium :+1:

**Comment by subins2000** [comment]
Yes, that'd be a good feature. But, then you'll have to rename the repo & project :P

"Quick Clone & Open With Any Editor"

**Pull request merged**